include/SFML/System/Angle.inl

```diff
@@ -0,0 +1,259 @@
+{
+    return left.asRadians() != right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator <(Angle left, Angle right)
+{
+    return left.asRadians() < right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator >(Angle left, Angle right)
+{
+    return left.asRadians() > right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator <=(Angle left, Angle right)
+{
+    return left.asRadians() <= right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator >=(Angle left, Angle right)
+{
+    return left.asRadians() >= right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator -(Angle right)
+{
+    return radians(-right.asRadians());
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator +(Angle left, Angle right)
+{
+    return radians(left.asRadians() + right.asRadians());
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle& operator +=(Angle& left, Angle right)
+{
+    return left = left + right;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator -(Angle left, Angle right)
+{
+    return radians(left.asRadians() - right.asRadians());
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle& operator -=(Angle& left, Angle right)
+{
+    return left = left - right;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator *(Angle left, float right)
+{
+    return radians(left.asRadians() * right);
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator *(float left, Angle right)
+{
+    return right * left;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle& operator *=(Angle& left, float right)
+{
+    return left = left * right;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator /(Angle left, float right)
+{
+    return radians(left.asRadians() / right);
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle& operator /=(Angle& left, float right)
+{
+    return left = left / right;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr float operator /(Angle left, Angle right)
+{
+    return left.asRadians() / right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle operator %(Angle left, Angle right)
+{
+    return radians(priv::positiveRemainder(left.asRadians(), right.asRadians()));
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle& operator %=(Angle& left, Angle right)
+{
+    return left = left % right;
+}
+
+inline namespace literals
```

Comment:
Two questions:
1. Why `inline`? I think typical usage will be `using namespace sf::literals;`, no?
2. This naming convention is inconsistent with `sf::Glsl`. Maybe let's keep `PascalCase` for now, and discuss change of naming convention in a separate issue (if that's desired).

Reply:
`inline` was @kimci86's idea. Apparently it allows us to use these literals within the `sf::` namespace without having to add that `using namespace sf::literals` declaration but still requires that users outside the `sf::` namespace do use that.

Are you saying that `literals::` should be `Literals::`? That's no problem to change.

Reply:
I went ahead and changed `literals::` to `Literals::`.

Reply:
I was inspired by standard library when suggesting `inline`.

Reply:
Are the symbols only available for SFML itself? I don't know if we want `using namespace sf;` to also spill them out.

Yes, `Literals::` for now. I'm also torn if it's better or not, but we should find a rule that holds for all namespaces 🙂
The thing is that there could be namespaces used like types, e.g. [`sf::Listener`](https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1Listener.php) might as well be a namespace.

Reply:
Turns out you're right. All it takes to get these UDLs is to add `using namespace sf`. I wasn't aware it worked that way but it makes more sense now. That's why we don't need that using declaration when inside the `sf::` namespace.

Reply:
Then I would say we make it a regular `namespace` instead of `inline namespace`. 

Users can still achieve the same, but usage of user-defined literals is explicit and not accidental. If we as SFML developers want to use them, we can still do so with selective `using` statements inside .cpp files or function bodies.

I would avoid that symbols appear in two different namespaces.

Reply:
I interpreted the standard library pattern as a good practice, but we are free to be a little more strict about namespaces. As you prefer.

Reply:
I removed `inline` from the `Literals` namespace.